sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ReceiverManager.cs

```diff
@@ -262,7 +262,8 @@ protected virtual async Task OnMessageHandler(ServiceBusReceivedMessage message,
                     // by the renewLockCancellationToken. This way we prevent a TaskCanceledException.
                     Task delayTask = await Task.Delay(delay, cancellationToken)
                         .ContinueWith(
-                            (t, s) => t,
+                            t => t,
```

Comment:
Gah, thanks for fixing this!

Would you like to add it to the change log under "Bugs Fixed"?

Reply:
Can do.